e2e/testcases/override_resource_limits_test.go

```diff
@@ -144,6 +149,7 @@ func TestOverrideReconcilerResourcesV1Alpha1(t *testing.T) {
 
 	// Verify ns-reconciler-frontend uses the default resource requests and limits
 	err = nt.Validate(core.NsReconcilerName(frontendNamespace, configsync.RepoSyncName), v1.NSConfigManagementSystem, &appsv1.Deployment{},
+		hasGeneration(nsReconcilerBackendDeployment.Generation),
```

Comment:
nsReconcilerBackendDeployment -> nsReconcilerFrontendDeployment

Reply:
Fixed it